Tests/NIOFileSystemIntegrationTests/FileHandleTests.swift

```diff
@@ -305,7 +305,11 @@ final class FileHandleTests: XCTestCase {
     func testWriteAndReadUnseekableFile() async throws {
         let privateTempDirPath = try await FileSystem.shared.createTemporaryDirectory(template: "test-XXX")
         self.addTeardownBlock {
-            try await FileSystem.shared.removeItem(at: privateTempDirPath, recursively: true)
+            try await FileSystem.shared.removeItem(
```

Comment:
The changes to add `.platformDefault` shouldn't be required as it's the default? (It's useful to _not_ make these changes to make sure we have an appropriate default.)

Reply:
I might have changes these tests after adding only some of the protocol extensions, as I've had those tests fail. Now with unnecessary implementations removed, tests are still passing. Going to revert these changes (including the format change—where I'm again surprised the Swift formatter isn't trying to remove the line breaks that are now not required).